src/NUnitFramework/framework/Constraints/RangeConstraint.cs

```diff
@@ -72,7 +87,7 @@ public override ConstraintResult ApplyTo<TActual>(TActual actual)
         {
             if ( from == null || to == null || actual == null)
                 throw new ArgumentException( "Cannot compare using a null reference", "actual" );
```

Comment:
@jnm2 I very strongly disagree. I understand what you mean and that it can be __thought__ of in that way, but a range is a concept, separate from any mathematical embodiment, which doesn't seem to me to have a place in it for null values. Anyway, it's good that we have pushed off the question to another discussion.